promql/testdata/native_histograms.test

```diff
@@ -0,0 +1,207 @@
+# Minimal valid case: an empty histogram.
+load 5m
+	empty_histogram{}	{{}}
+
+eval instant at 5m histogram_count(empty_histogram{})
```

Comment:
The empty `{}` for label selections aren't required. I would usually remove them.

Reply:
Removed all empty label selectors in 3eb0735a7.